pkgs/development/tools/lazygit/default.nix

```diff
@@ -1,27 +1,24 @@
 { stdenv, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
-  name = "lazygit-${version}";
-  version = "0.7.2";
+  pname = "lazygit";
+  version = "0.8";
 
   goPackagePath = "github.com/jesseduffield/lazygit";
 
+  subPackages = [ "." ];
+
   src = fetchFromGitHub {
     owner = "jesseduffield";
-    repo = "lazygit";
+    repo = pname;
     rev = "v${version}";
-    sha256 = "1b5mzmxw715cx7b0n22hvrpk0dbavzypljc7skwmh8k1nlx935jj";
+    sha256 = "0zynw5gr96a59x1qshzhhvld883ndf1plnw6l9dbhmff0wcfv6l1";
   };
 
-  postPatch = ''
-    rm -rf scripts
-  '';
-
   meta = with stdenv.lib; {
-    inherit (src.meta) homepage;
     description = "Simple terminal UI for git commands";
+    homepage = "https://github.com/jesseduffield/lazygit";
     license = licenses.mit;
-    maintainers = with stdenv.lib.maintainers; [ fpletz ];
-    platforms = stdenv.lib.platforms.unix;
```

Comment:
why the removal ?

Reply:
Seems to also work on windows, since is has [releases for it](https://github.com/jesseduffield/lazygit/releases/tag/v0.8.1); and [`buildGoPackage` defaults this to `plataform.all`](https://github.com/nixos/nixpkgs/blob/0d1707c83468ba6b07d802e5ed097c1789585950/pkgs/development/go-packages/generic/default.nix#L235)